config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -194,6 +194,9 @@ presets:
   # Increase delete collection parallelism.
   - name: TEST_CLUSTER_DELETE_COLLECTION_WORKERS
     value: --delete-collection-workers=16
+  # Disable kubernetes-dashboard
```

Comment:
It's under preset-e2e-scalability-node: "true". I think you wanted to put this into some common preset, right?